lib/chef/resource/windows_certificate.rb

```diff
@@ -87,6 +87,10 @@ class WindowsCertificate < Chef::Resource
         description: "Ensure that sensitive resource data is not logged by the #{ChefUtils::Dist::Infra::CLIENT}.",
         default: lazy { pfx_password ? true : false }, skip_docs: true
 
+      property :exportable, Integer,
+        description: "Ensure that imported pfx certificate is exportable. Please provide '1' if you want the certificate to be exportable."
+        default: 0
+
       action :create do
```

Comment:
I think this would make more sense to a user if it was true/false instead of an integer.

Reply:
Updated! Thank you @mwrock